salt/returners/mysql.py

```diff
@@ -198,7 +200,8 @@ def _get_options(ret=None):
                 'port': 3306,
                 'ssl_ca': None,
                 'ssl_cert': None,
-                'ssl_key': None}
+                'ssl_key': None,
+                'unix_socket': None}
```

Comment:
Above it says that `/tmp/mysql.sock` is the default but here it appears to be `None`?

Reply:
good point, i think the default can safely be set here, because both [MySQLdb](https://github.com/dccmx/mysqldb/blob/master/doc/MySQLdb.txt#L137) and [PyMySQL](https://github.com/PyMySQL/PyMySQL/blob/master/pymysql/connections.py#L944) will only use a `unix_socket` when host == localhost

so the default `unix_socket` will just be discarded for someone who wants to use a remote connection